spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/BigQuerySQLQuery.scala

```diff
@@ -67,7 +68,11 @@ abstract class BigQuerySQLQuery(
     if (fields.isEmpty) {
       children.foldLeft(Seq.empty[Attribute])(
         (x, y) => {
-          val attrs = y.outputWithQualifier
+          val attrs = if (visibleAttribute.isEmpty) {
```

Comment:
Shouldn't we have `if (y.visibleAttributes.isEmpty)` here and `y.visibleAttribute.get` on line 74?

Reply:
Addressed